geonode/settings.py

```diff
@@ -21,62 +21,75 @@
 # Django settings for the GeoNode project.
 import os
 from kombu import Queue
-import geonode
+from geonode import __file__ as geonode_path
+from geonode import get_version
 from geonode.celery_app import app  # flake8: noqa
+import djcelery
+import dj_database_url
+
+
+def str2bool(v):
```

Comment:
instead of creating `str2bool` here to try & anticipate all the varieties of "True" that might be used in settings files, why don't we just drop this custom handling & just enforce consistency from the start (i.e., comparing some_settings_value == "True")